### PR TITLE
Feature/nav clickable parent

### PR DESCRIPTION
**Pull request by Akswii**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 ### Changed
 
+- Updated nav styling to fit Swedbank Pay better.
```
Comment (hmosvoll) [inline CHANGELOG.md]: Is this all you have done in this branch?

Comment (Akswii) [reply CHANGELOG.md]: Roughly, yes. Should I outline all the changes to nav i've made? They're all visual as far as i can tell. Except maybe clicking the parent makes the child active, but this is not included functionality when someone uses the nav, it only works for our documentation.

**Comment by Akswii** [comment]
> 
> 
> Why is it the active state not shown on mobile?
> Why are the "Transactions" and "Account" items on "Two levels" disappearing on mobile?
> The CSS for this component seems to have many selector with deep dependency on very common elements such as `<span>`, `<i>` and `<a>`, without any child combinator (`>`). I suspect this might be the reason for some of the frustration surrounding this component.

Do you want me to try to clean up the component? As for the two menu items dissapearing that is due to them containing child elements. We should discuss this face to face.

**Comment by hmosvoll** [comment]
> > Why is it the active state not shown on mobile?
> > Why are the "Transactions" and "Account" items on "Two levels" disappearing on mobile?
> > The CSS for this component seems to have many selector with deep dependency on very common elements such as `<span>`, `<i>` and `<a>`, without any child combinator (`>`). I suspect this might be the reason for some of the frustration surrounding this component.
> 
> Do you want me to try to clean up the component? As for the two menu items dissapearing that is due to them containing child elements. We should discuss this face to face.
Sure.

File: CHANGELOG.md (hunk 1 above)
Comment (hmosvoll) [reply CHANGELOG.md]: Is not visual changes just as important for users of the DG? 
My point is; when I read this I don't really know what changed. 
What can I expect if I update the DesignGuide? 

First part: Something changed in nav
Second: To fit Swedbank Pay, is this not the reason for almost all our changes?

Comment (Akswii) [reply CHANGELOG.md]: Yes that is the reason for all our changes, but the nav hasn't really been handled so far in the rebranding. I'm just not entirely sure what you want me to include in the docs. We should discuss this face to face aswell.

**Pull request merged**